app/controllers/games_controller.rb

```diff
@@ -23,6 +23,17 @@ def update
     end
   end
 
+  def destroy
+    games = Game.all
+    game = Game.find(params[:id])
+    if game.destroy
+      render json: game
```

Comment:
Was the goal to render all the `games` available? Or the `game` that was destroyed?

Reply:
I think I think I had it there is to render all the games so we know the delete one is gone =O but I probably don't need that since I have test for it